packages/combo-box/src/vaadin-combo-box-data-provider-mixin.js

```diff
@@ -224,19 +224,17 @@ export const ComboBoxDataProviderMixin = (superClass) =>
         const { rootCache } = this.__dataProviderController;
         if (rootCache.items !== items) {
           rootCache.items = items;
-          this.requestContentUpdate();
+          this.synchronizeControllerState();
         }
       }
     }
 
     /**
      * Synchronizes the controller's state with the component, which can be
      * out of sync after the controller receives new data from the data provider
-     * or if the state in the controller is directly manupulated.
-     *
-     * @override
+     * or if the state in the controller is directly manipulated.
      */
-    requestContentUpdate() {
+    synchronizeControllerState() {
```

Comment:
Let's keep this method private.

```suggestion
    __synchronizeControllerState() {
```

Reply:
Done